README.md

```diff
@@ -13,6 +13,7 @@
 - [Articles](#articles)
 - [Task Runners](#task-runners)
 - [File Watchers](#file-watchers)
+- [Dev Servers](#doc-servers)
```

Comment:
I tried to run use `npm run doctoc` but for a reason I can't find, it always come back to the state before my edit, even after having manually edited the ToC oO